lib/reference.coffee

```diff
@@ -131,11 +133,20 @@ class Reference
 
   # Get the number of verses in the given book id
   @versesInBookId: (bookId) ->
-    return books[bookId].verses.reduce((a, b) -> a + b)
+    return books[bookId-1].verses.reduce (a, b) -> a + b
+
+  # Get the number of verses in the given chapter id
+  @versesInChapterId: (chapterId) ->
+    chapter = 1
+    for book in books
+      for verse in book.verses
+        return verse if chapter is chapterId
```

Comment:
This essentially counts up until you reach the chapter ID in question. A faster approach would be to count the number of chapters in a book (i.e. `book.length`) and subtract that from the id as you go, rather than iterating through each chapter of each book. Something like:

```coffee
for book in books
  if chapterId > books.length
    chapterId -= books.length
  else
    return books[chapterId]
# potentially throw an error or something here - this line outside the for loop 
# should be impossible to reach for valid references
```